internal/js/modules/k6/browser/common/http.go

```diff
@@ -718,6 +718,14 @@ type Route struct {
 	handled bool
 }
 
+// ContinueOptions are request fields that can be overridden when continuing a request.
+type ContinueOptions struct {
+	Headers  []HTTPHeader
+	Method   string
+	PostData string
```

Comment:
This likely should be `[]byte` and get converted from string or extracted from an ArrayBuffer. 

See https://pkg.go.dev/go.k6.io/k6/js/common#ToBytes

Reply:
Why is that @mstoykov? I'd like to understand too incase i've made a mistake elsewhere.

Reply:
From https://playwright.dev/docs/api/class-route#route-continue - it can take binary data as input as ArrayBuffer.

If you try to safe binary data as string .. it might work, or it might mangle it a little bit (or a lot). As such in general we prefer to keep it as a slice of bytes. Sobek/goja (used to(?))keeps some string as UTF16 for compatibility with the standard. So in a lot of the cases we will have copy/conversion either way.

Reply:
Updated in https://github.com/grafana/k6/pull/4971/commits/05f4c7e2edff106981338efdbfd84328ea2d9a15 :+1: If it looks good, I'll do the same for the `fulfill` PR.

Reply:
Thanks for bringing this topic @mstoykov, as at some point while reviewing this pull request, I wondered if `string` would be good for all cases, but then I forgot to express my doubts.

Yeah, I think that if we speak about data in an HTTP POST request's body, it's safer to use raw/bytes data, as I'd expect the transformation from `string` to `bytes`(binary) be always safe but not the other way round, as you said.